README.md

```diff
@@ -0,0 +1,121 @@
+
+## Features
+
+-   Efficient Majority Judgment algorithm, scales well to billions of participants
+-   Configure whether to favor _adhesion_ or _contestation_ (default)
+-   Balance proposal tallies using a static default grade or the median grade
+-   Room for Central Judgment and Usual Judgment
+-   Unit-tested (run `npm run converage`)
+
+## Install
+
+`npm install majority-judgment`
+
+## Get started
+
+### Random sample
+
+```typescript
+import {
+    TallyCollector,
+    MajorityJudgmentDeliberator,
+    IDeliberator,
+    IResult,
+} from "majority-judgment";
+
+const proposalAmount: number = 5;
+const mentionAmount: number = 7;
+const voterAmount: number = 1000;
+const tally: TallyCollector = new TallyCollector(proposalAmount, mentionAmount);
+fillTallyCollectorWithRandomVote(tally, voterAmount);
+
+const deliberator: IDeliberator = new MajorityJudgmentDeliberator();
+const result: IResult = deliberator.deliberate(tally);
+
+for (let i: number = 0; i < proposalAmount; i++)
+    console.log(
+        `Proposal at index ${i} obtains the rank ${result.proposalResults[i].rank} with the majority mention at index ${result.proposalResults[i].analysis.medianMentionIndex}`
+    );
+
+function fillTallyCollectorWithRandomVote(tally: TallyCollector, voterAmount: number): void {
+    const applyRandomVoteForEachProposal = () => {
+        let randomProposalIndex: number;
+        let randomMentionIndex: number;
+
+        for (let i: number = 0; i < proposalAmount; i++) {
+            randomProposalIndex = Math.floor(Math.random() * proposalAmount);
+            randomMentionIndex = Math.floor(Math.random() * mentionAmount);
+            tally.collect(randomProposalIndex, randomMentionIndex);
+        }
+    };
+
+    for (let i: number = 0; i < voterAmount; i++) applyRandomVoteForEachProposal();
+}
+```
+
+### Knewn datas
+
+```typescript
+import {
+    MajorityJudgmentDeliberator,
+    IDeliberator,
+    IResult,
+    ITally,
+    Tally,
+    Proposal,
+} from "majority-judgment";
+
+const meritProfilSample: bigint[] = [4n, 0n, 2n, 1n];
+// index 0 = worst mention, 4 vote
+// max index (3) = best mention, 1 vote
+
+const tally: ITally = new Tally([
+    new Proposal(meritProfilSample),
+    new Proposal([2n, 1n, 0n, 4n]),
+    new Proposal([3n, 2n, 1n, 1n]),
+]);
+
+const deliberator: IDeliberator = new MajorityJudgmentDeliberator();
+const result: IResult = deliberator.deliberate(tally);
+const proposalAmount: number = tally.proposalAmount;
+
+for (let i: number = 0; i < proposalAmount; i++)
+    console.log(
+        `Proposal at index ${i} obtains the rank ${result.proposalResults[i].rank} with the majority mention at index ${result.proposalResults[i].analysis.medianMentionIndex}`
+    );
+
+// Note: the sum vote in proposal must be equal. If not, the deliberator will throw an error.
+```
+
+### Prevent error when vote amount is not the same for each proposal
```

Comment:
Again the purpose of this example is to manage cases with an irregular number of vote per proposal, not about hiding an error message. So we should be clearer and call this example "Majority judgment where voters did not vote for every proposal". We should also add a paragraph below like this one:

In some deliberations, the number of proposals can be very high -- think for example about participatory budgeting. In this case, the voter might rather some votes for a sample of proposals, and left other blanked. We provide the `NormalizedTally` to manage this case. Note that in such deliberation, the proposals should be randomly placed on a ballot sheet, to avoid highly under sampled proposals.